boards/arm/stm32f072b_disco/pinmux.c

```diff
@@ -32,6 +32,11 @@ static const struct pin_config pinconf[] = {
 	{STM32_PIN_PB4, STM32F0_PINMUX_FUNC_PB4_SPI1_MISO},
 	{STM32_PIN_PB5, STM32F0_PINMUX_FUNC_PB5_SPI1_MOSI},
 #endif /* CONFIG_SPI_1 */
+#ifdef CONFIG_CAN_1
+   {STM32_PIN_PB8, STM32F0_PINMUX_FUNC_PB8_CAN_RX},
```

Comment:
Indentation seems to be wrong here

Reply:
Sorry for this huge bunch of styling issues. I should have checked this before submitting. Shame on me :-)